src/networks.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "name": "ropsten",
+    "rpcUrl": "https://ropsten.infura.io/ethr-did",
```

Comment:
these are old infura URLs and don't use the new project based address.

I don't think we should be publishing these defaults.. or at least not as is.